tests/test_likelihoods.py

```diff
@@ -61,16 +64,44 @@ def __repr__(self):
 likelihood_setups = [
     LikelihoodSetup(Gaussian()),
     LikelihoodSetup(StudentT()),
-    LikelihoodSetup(Beta()),
-    LikelihoodSetup(MultiClass(2), Y=tf.argmax(Datum.Y, 1).numpy().reshape(-1, 1), rtol=1e-3, atol=1e-3),
-    LikelihoodSetup(Ordinal(np.array([-1, 1])), Y=np.random.randint(0, 3, Datum.Yshape)),
-    LikelihoodSetup(Poisson(invlink=Datum.square)),
-    LikelihoodSetup(Exponential(invlink=Datum.square)),
-    LikelihoodSetup(Gamma(invlink=Datum.square)),
-    LikelihoodSetup(Bernoulli(invlink=tf.sigmoid)),
+    LikelihoodSetup(Beta(),
+                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    LikelihoodSetup(Ordinal(np.array([-1, 1])),
+                    Y=tf.random.uniform(Datum.Yshape, 0, 3, dtype=default_int())),
+    LikelihoodSetup(Poisson(invlink=tf.square),
+                    Y=tf.random.poisson(Datum.Yshape, 1.0, dtype=default_float())),
+    LikelihoodSetup(Exponential(invlink=tf.square),
+                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    LikelihoodSetup(Gamma(invlink=tf.square),
+                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
+    LikelihoodSetup(Bernoulli(invlink=tf.sigmoid),
+                    Y=tf.random.uniform(Datum.Yshape, dtype=default_float())),
```

Comment:
I meant this block of code, but now I can see that you changed it. You could create two random variables and reuse them in likelihood setup. It would be also useful for debugging in the future and the code would be more readable.

Reply:
I could, but then we'd need four different `Y`s in the `Datum` class, which I don't think is an improvement over the status quo. If you have a better way of doing it in mind, please feel free to change it, maybe you could do that in a different PR?